### ISSUE TITLE
White font colour with white background

### ISSUE THREAD

**Issue by gonzalodeniz** [nilsteampassnet/TeamPass#4374]
### Steps to reproduce
1. Switch to dark theme
2. Check/uncheck the box for ‘Limited search’ until the background of the search box is white.
3. Type in the search box.

### Expected behaviour
We should see the letters in black

### Actual behaviour
![image](https://github.com/user-attachments/assets/e016d56d-66d0-454e-873d-64f2d672d501)
![image](https://github.com/user-attachments/assets/ea5b27cd-6c2f-44e0-963b-0aeb9e01b633)

**Teampass version:**  3.1.2.120

**Comment by corentin-soriano** [comment]
I understand what happen. In `pages/items.js.php`:

```php
if ($(this).is(":checked") != "<?php echo $SETTINGS['limited_search_default']; ?>") {
            $('#find_items').css({
                "background-color": "#f56954"
            });
        } else {
            $('#find_items').css({
                "background-color": "#FFF"
            })
        }
    });
```
Week-end must change .css() to something like:

`$('#find_items').addClass('bg-red');`
And remove in else:
`$('#find_items').addClass('bg-red');`

I will fix that issue next week.

**Comment by gonzalodeniz** [comment]
Thank you very much Corentin-Soriano.

 If you have time could you please look at issue #4316? I think the solution is simple.

**Comment by gonzalodeniz** [comment]
Fixed in v3.1.2.130
